### PR TITLE
feat: Test the `otherMarketData.json` file against a live slinky instance

### PR DESCRIPTION
**Pull request by nivasan1**
## In This PR
- I migrate the version of v4-chain used for testing the `otherMarketData.json` to [here](https://github.com/dydxprotocol/v4-chain/pull/1452)
    - This is the skip-fork of the v4-chain binary, we will change this to mainline once [this](https://github.com/dydxprotocol/v4-chain/pull/1452) is merged
- I force the `.github/workflows/validate-other-market-data.yml` workflow to trigger on any changes to `scripts/validate-other-market-data.ts`
- I add additional tests for each price-feed's health in accordance with slinky

### CODE REVIEW

File: .github/workflows/validate-other-market-data.yml [hunk 1]
```diff
@@ -35,8 +36,8 @@ jobs:
       - name: Checkout v4-chain repo
         uses: actions/checkout@v3
         with:
-          repository: 'dydxprotocol/v4-chain'
-          ref: '7604f974a699efa8d15a8a70bfc85d8b5b2e2d58'
+          repository: 'skip-mev/v4-chain'
```
Comment (nivasan1) [inline .github/workflows/validate-other-market-data.yml:39]: this tag is pointing to a skip fork temporarily, pending [this](https://github.com/dydxprotocol/v4-chain/pull/1452) PR

File: scripts/validate-other-market-data.ts [hunk 2]
```diff
@@ -112,10 +114,16 @@ enum ExchangeName {
   Okx = 'Okx',
 }
 
+interface PrometheusTimeSerie {
```
Comment (Christopher-Li) [inline scripts/validate-other-market-data.ts:117]: nit: time series?

File: scripts/validate-other-market-data.ts [hunk 3]
```diff
@@ -389,10 +412,10 @@ async function validateAgainstLocalnet(proposals: Proposal[]): Promise<void> {
...
             subticksPerTick: proposal.params.subticksPerTick,
             delayBlocks: proposal.params.delayBlocks,
-            // marketType:
-            //   proposal.params.marketType === 'PERPETUAL_MARKET_TYPE_ISOLATED'
-            //     ? PerpetualMarketType.PERPETUAL_MARKET_TYPE_ISOLATED
-            //     : PerpetualMarketType.PERPETUAL_MARKET_TYPE_CROSS,
+            marketType:
+              proposal.params.marketType === 'PERPETUAL_MARKET_TYPE_ISOLATED'
+                ? PerpetualMarketType.PERPETUAL_MARKET_TYPE_ISOLATED
+                : PerpetualMarketType.PERPETUAL_MARKET_TYPE_CROSS,
```
Comment (Christopher-Li) [inline scripts/validate-other-market-data.ts:418]: @jaredvu are we ready to uncomment this?

Comment (jaredvu) [reply scripts/validate-other-market-data.ts:418]: We aren't totally ready to uncomment this yet. We want to make sure the json is compatible with v4.0

File: scripts/validate-other-market-data.ts [hunk 4]
```diff
@@ -14,13 +14,12 @@ import { BroadcastTxSyncResponse } from '@cosmjs/tendermint-rpc/build/tendermint
...
-  Network,
-  ProposalStatus,
-  TransactionOptions,
+  Network, TransactionOptions,
   VoteOption,
+  ProposalStatus
 } from '@dydxprotocol/v4-client-js';
 import {
-  Perpetual, // PerpetualMarketType,
+  Perpetual, PerpetualMarketType,
```
Comment (jaredvu) [inline scripts/validate-other-market-data.ts:22]: nit: cleanup import formatting

**Review by jaredvu — APPROVED** [review]

**Pull request merged**